tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -89,9 +89,12 @@ def only_register(callback_fn, module, overwrite):  # pylint: disable=unused-arg
 
 
 @pytest.fixture(autouse=True)
-def disable_new_import_hook_firing_if_module_already_exists():
-    with mock.patch("mlflow.tracking.fluent.register_post_import_hook", wraps=only_register):
+def disable_new_import_hook_firing_if_module_already_exists(request):
+    if "do_not_disable_new_import_hook_firing_if_module_already_exists" in request.keywords:
```

Comment:
Why is this necessary?

Reply:
Nevermind, I see.

Reply:
Recall that offline I mentioned how `mlflow.last_active_run` works in a notebook environment, but doesn't in test. The reason is because by mocking out `mlflow.tracking.fluent.register_post_import_hook`, the `_active_run_stack` in `fluent.py` does not get populated.